addons/payment_mercado_pago/models/payment_transaction.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import pprint
 
+from urllib.parse import quote as url_quote
 from werkzeug import urls
```

Comment:
`urllib` is a standard lib so it can go in the first group.
```suggestion
import pprint
from urllib.parse import quote as url_quote

from werkzeug import urls
```

Reply:
fixed